test/test_other.py

```diff
@@ -7556,10 +7556,11 @@ def test_override_c_environ(self):
     output = self.run_js('run.js')
     self.assertContained('|world|', output)
 
+  @also_with_wasmfs
   def test_warn_no_filesystem(self):
     error = 'Filesystem support (FS) was not included. The problem is that you are using files from JS, but files were not used from C/C++, so filesystem support was not auto-included. You can force-include filesystem support with -sFORCE_FILESYSTEM'
 
-    self.run_process([EMCC, test_file('hello_world.c')])
+    self.emcc(test_file('hello_world.c'))
```

Comment:
Another way to do this would be to do ` + self.get_emcc_args()`.. I'm not sure I have a preference.. but maybe one day we should make this consistent.

Reply:
I have a slight preference for the shorter form (`self.emcc`) I think.